core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -394,6 +394,14 @@ static RelDataType deriveTypeSplit(SqlOperatorBinding operatorBinding,
   public static final SqlFunction SUBSTR_POSTGRESQL =
       SUBSTR.withKind(SqlKind.SUBSTR_POSTGRESQL);
 
+  /** The "PARSE_URL(urlString, partToExtract [, keyToExtract])" function. */
```

Comment:
I found that both with and without spaces were used in the previous code, so I want to know which one is recommended and would you like to explain the reason?

Reply:
Yeah that's true there are inconsistencies. I think it makes sense to have spaces on either both sides or neither. Since nearly all examples have a space (`foo [, bar`) at the beginning it makes sense to match. But you are right it is not consistent across all documentation.

Reply:
Fixed.